bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -85,6 +85,8 @@ func GetKfApp(kfdef *kfdefsv2.KfDef) kftypes.KfApp {
 	return _coordinator
 }
 
+// Config itself is a kustomize package, https://github.com/kubeflow/kubeflow/tree/master/bootstrap/config.
+// getConfigFromCache returns the KfDef yaml (in bytes) with overlays applied, and also writes it to app.yaml.
```

Comment:
nit reverse the order of the two lines of the comment so that the first line in the comment is "getConfigFromCache..."

Reply:
updated, thanks